### DISCUSSION TITLE
Welcome to DNNV Discussions!

### DISCUSSION THREAD

**Discussion by dlshriver** [dlshriver/DNNV#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with others.

  To get started, comment below with an introduction of yourself and how you use DNNV.

**Comment by RahmaAbdessamad** [comment]
Hello my name is Rahma Abdessamad and I am working on a project of a neural network verification to obtain my master's degree. For that I'll be using DNNV. I am trying to install it using the commands in the documentation. I had so much errors but I could resolve it. Now I don't get any error but also I don't know how to make sure that dnnv is well installed.

**Reply by dlshriver** [reply]
What seems to be the issue? Have you tried using it? Does the command `dnnv -h` give output similar to that shown in the [documentation](https://docs.dnnv.org/en/stable/getting_started/basic_usage.html)?

**Reply by RahmaAbdessamad** [reply]
Yes it gives the same output. That means it was installed ?

**Reply by dlshriver** [reply]
If it is not producing errors and is giving the same output as the documentation, then it is likely installed correctly.

**Reply by RahmaAbdessamad** [reply]
Great ! Thank you so much for helping
